test/client/doc.js

```diff
@@ -214,6 +214,45 @@ describe('Doc', function() {
       });
     });
 
+    it('remote ops are transformed by ops submitted in `before op` event handlers', function(done) {
+      var doc = this.doc;
+      var doc2 = this.doc2;
+      var doc3 = this.doc3;
```

Comment:
Unused

Reply:
it's used in `verifyConsistency`.

Reply:
Oh yes I forgot we had to pass all the docs in there. Bit weird, but hey 🤷🏼